test/sciml_style.jl

```diff
@@ -1434,4 +1492,500 @@
+        setup = Pair{String, Any}["Start time" => first(integrator.sol.prob.tspan),
+                                  "Final time" => last(integrator.sol.prob.tspan),
+                                  "time integrator" => integrator.alg |> typeof |> nameof,
+                                  "adaptive" => integrator.opts.adaptive]
+        """
+        @test format_text(str, SciMLStyle()) == expected
+
+        # Test simpler typed arrays
+        str2 = raw"""
+        values = Int[1,
+                     2,
+                     3]
+        """
+        expected2 = raw"""
+        values = Int[1,
+                     2,
+                     3]
+        """
+        @test format_text(str2, SciMLStyle()) == expected2
+
+        # Another alignment test
+        str3 = raw"""
+        x = Float64[1.0,
+                    2.0,
+                    3.0]
+        """
+        expected3 = raw"""
+        x = Float64[1.0,
+                    2.0,
+                    3.0]
+        """
+        @test format_text(str3, SciMLStyle()) == expected3
+    end
+
+    @testset "Issue #934 comment #3146379860 - Additional edge cases" begin
+        # Test 1: Variable array indent
+        str = raw"""
+        kernels = [
+            GaussianKernel,
+            SchoenbergCubicSplineKernel,
+            SchoenbergQuarticSplineKernel,
+            SchoenbergQuinticSplineKernel,
+        ]
+        """
+        # With YAS-style alignment, arrays align with opening bracket
+        expected = raw"""
+        kernels = [GaussianKernel,
+                   SchoenbergCubicSplineKernel,
+                   SchoenbergQuarticSplineKernel,
+                   SchoenbergQuinticSplineKernel]
+        """
+        # Default style now uses YAS-style alignment
+        @test format_text(str, SciMLStyle()) == expected
+        
+        # With yas_style_nesting=true both versions should be valid and not be reformatted
+        @test format_text(str, SciMLStyle(), yas_style_nesting = true) == str
+        @test format_text(expected, SciMLStyle(), yas_style_nesting = true) == expected
+        
+        # Test 1b: Bad array alignment (from efaulhaber's comment #3148497419)
+        # With PR #807, existing line breaks are preserved with standard indentation
+        str_bad = raw"""
+        kernels = [
+                   GaussianKernel,
+                   SchoenbergCubicSplineKernel,
+                   SchoenbergQuarticSplineKernel,
+                   SchoenbergQuinticSplineKernel
+                   ]
+        """
+        # Should preserve line breaks with standard indentation
+        expected_fixed = raw"""
+        kernels = [GaussianKernel,
+            SchoenbergCubicSplineKernel,
+            SchoenbergQuarticSplineKernel,
+            SchoenbergQuinticSplineKernel]
+        """
+        # The line break should be preserved and indent should be changed to 4 spaces
+        @test format_text(str_bad, SciMLStyle(), yas_style_nesting = true) == expected_yas
```

Comment:
```suggestion
        str_bad2 = raw"""
        kernels = [GaussianKernel,
            SchoenbergCubicSplineKernel,
            SchoenbergQuarticSplineKernel,
            SchoenbergQuinticSplineKernel]
        """
        # The line break should be preserved and indent should be changed to 4 spaces
        @test format_text(str_bad, SciMLStyle(), yas_style_nesting = true) == str
        # The line break should be preserved and indent should align with opening paren
        @test format_text(str_bad2, SciMLStyle(), yas_style_nesting = true) == expected
```
Maybe also rename `str` to `expected_variable` to make this clearer?